src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/UnnecessarySemicolonInEnumeration/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="UnnecessarySemicolonInEnumeration"/>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="context" />
+            <property name="neverSuppressedChecks" value="UnnecessarySemicolonInEnumerationCheck" />
```

Comment:
Please explain why do you need this property here and what's the problem not using it at all or using another property instead?

Reply:
as the case mentioned in https://github.com/checkstyle/patch-filters/pull/183#issue-458262492, 
```
@@ -23,7 +23,6 @@ public class Test {
         A,
         B,
         ;  // violation without filter
-        six(){}
     }
```
`six(){}` cause the violation, but it is not child node of ` ;`, so if we do not use `neverSuppressedChecks`, the violation above will be suppressed.

Reply:
Can you use `checkNamesForContextStrategyByTokenOrParentSet` instead?

Reply:
@rdiachenko I try it and it's ok. I have change it to `checkNamesForContextStrategyByTokenOrParentSet`. thanks.